### PR TITLE
created json branch

### PR DESCRIPTION
**Pull request by ItsMina7**

### CODE REVIEW

**Comment by velomash** [comment]
Hey! This is amazing! You just made your first pull request.

> Pull requests let you tell others about changes you've pushed to a GitHub repository. Once a pull request is sent, interested parties can review the set of changes, discuss potential modifications, and even push follow-up commits if necessary.
> https://yangsu.github.io/pull-request-tutorial/

If you start every major change to your project with a feature branch (multiple-questions in this case) then we review your updates with a pull request then you'll never mess up a project! It's impossible.

This pull request is basically saying, lets review these changes before adding them to master. Once we approve the request all our new changes will be in master.

File: src/multipleQuestions.json
Comment (velomash) [inline src/multipleQuestions.json]: Great job adding this new file. Looks like we need to add some questions into this JSON. Luckily, we already have a template for what a question looks like. Copy the question data object from question.js, and use that structure to make three questions in this JSON.

**Review by velomash — CHANGES_REQUESTED** [review]
These questions are great! Excellent job adding stuff into this file. There's one thing though. You have commas separating the question objects, but the objects aren't in a list structure. Lets wrap this whole thing in an array so our JSON file is an array of question objects.

**Comment by ItsMina7** [comment]
I put everything in an array!

**Review by velomash — APPROVED** [review]

**Comment by velomash** [comment]
Hey great! I think this pull request is done. Go ahead and merge the pull request by hitting the green merge button.

**Pull request merged**
